### ISSUE TITLE
New Error - 'HF Token'

### ISSUE THREAD

**Issue by Pochoxo** [meizhong986/WhisperJAV#2]
Hello,

I tried using v0.6i today and it gave me this error when trying to run Whisper on the audio files in MyDrive:

/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_token.py:72: UserWarning: 
The secret `HF_TOKEN` does not exist in your Colab secrets.
To authenticate with the Hugging Face Hub, create a token in your settings tab (https://huggingface.co/settings/tokens), set it as secret in your Google Colab and restart your session.
You will be able to reuse this secret in all of your notebooks.
Please note that authentication is recommended but still optional to access public models or datasets.
  warnings.warn

Any help would be much appreciated.  Thanks and happy new year!

**Comment by meizhong986** [comment]
It seems that the warning that you receive is not the cause of the problem, but the cuda incompatibility after google upgraded colab. 

https://github.com/googlecolab/colabtools/issues/4214

It seems that the fallback method is not working any longer in colab. I will need to find a workaround.

**Comment by meizhong986** [comment]
For now version 1.0 BETA is a quick workaround. Please check it out to see that it works in your env.  
Note: the speed is not as fast as 0.6. We have to wait for the root cause problem to be solved before going back to the 2x speed.

**Comment by meizhong986** [comment]
Version 0.7 was just released. This release has the fix for the break after the colab upgrade. No need for the fallback runtime in this release.

**Comment by Pochoxo** [comment]
I just tried the new 0.7 and it works great.  The Colab upgrade issue has been fixed and the hallucination detection seems to work even better than before.  I just tried about 10 files and there were no hallucinations left in any of them.  Awesome job and thank you for the quick fix!!

On a side note, would it be possible to translate .m4a files in addition to .mp3 files?  Using FFMPEG to extract .m4a files is much faster but it doesn't look like it's a recognized audio file when I run the Colab workbook.  Not sure if this is an issue with the workbook or a limitation of Whisper so thought I'd ask.  Thank you again!
